### PR TITLE
New language: Marathi

### PR DESCRIPTION
**Pull request by prem-k-r**

### CODE REVIEW

**Comment by prem-k-r** [comment]
@ceskyDJ please review czech translation

**Comment by XengShi** [comment]
Great, I was also thinking about adding other Indian languages but not getting any requests or contributors (since most of us uses English!)

**Comment by prem-k-r** [comment]
It's ready just that Poppins doesn't support Devanagari numerals, have to use Noto Sans Devanagari for it.

**Comment by ceskyDJ** [comment]
> @ceskyDJ please review czech translation

I know, I'm so sorry, I was too much busy. I'll check it today.

**Comment by prem-k-r** [comment]
> > @ceskyDJ please review czech translation
> 
> I know, I'm so sorry, I was too much busy. I'll check it today.

No worries, I have done it in this pr, using your previously translated texts for AI-Tools. You can improve them whenever you are free, no hurry.

**Comment by XengShi** [comment]
image

filterFavicon is never used. If it is not being used, comment it out or remove it.

**Comment by XengShi** [comment]
For feels like you can keep just 'Mehsoos' instead of 'Mehsoos Hoto' to make it look better

**Comment by prem-k-r** [comment]
> For feels like you can keep just 'Mehsoos' instead of 'Mehsoos Hoto' to make it look better

Changed to `वाटते`, better choice

File: languages.js [hunk 1]
```diff
@@ -406,8 +406,8 @@ const translations = {
         "adaptiveIconInfoText": "Ikony zkratek se zmenší",
         "ai_tools_button": "AI nástroje",
         "enable_ai_tools": "Zobrazí zkratky AI nástrojů",
-        "googleAppsMenuText": "Google Apps",
-        "googleAppsMenuInfo": "Show shortcut for Google Apps",
+        "googleAppsMenuText": "Google aplikace",
+        "googleAppsMenuInfo": "Zobrazí zkratky Google aplikace",
```
Comment (ceskyDJ) [inline languages.js:410]:
```suggestion
        "googleAppsMenuInfo": "Zobrazí zkratky Google aplikací",
```

**Comment by prem-k-r** [comment]
Numerals showing in Android-Chrome 🫠
IMG_20241123_003301

Also, 9-dot icon seems not to be properly responsive. Top is fine, but right side is not fixed.

**Pull request merged**